### DISCUSSION TITLE
Tuya TS011F_plug_1 is reporting wrong power values

### DISCUSSION THREAD

**Discussion by Oulewappeur** [Koenkk/zigbee2mqtt#17120] [Devices]
Hi,

I've recently purchased three Aubess 20A Smartplugs (they Identify as Tuya TS011F_plug_1).

2 out of 3 work as expected. These report the correct values of energy & power.

One of them is having troubles, I discovered this while it was connected to my dishwasher, where it started to report power peaks of 20.000 W! The result was that the plug turned itself off automatically when this occured. Also connecting it to a 20W phone adapter gave measurements of 100W, but since that is within the threshold, it didn't turn itself off. 

I've updated the firmware to the most recent version (on all of the plugs), but still that doesn't help.

I'm trying to figure out if I can recalibrate the plug, but I have no clue how to do this.

Hopefully someone can help me out!

Cheers,

**Comment by arturmietek** [comment]
I also have problem with power and current values that stays 0 for TS011F_din_smart_relay (ATMS1602Z, _TZ3000_ky0fq4ho) from version 1.32.1. Before update everything works perfect also with HA automation on defined power value)


Simultaneously TS011F_plug_1 (_TZ3000_gjnozsaz) works perfect. Power and current readings are OK.

I had hopes that version 1.32.2 would bring fix but no :(

**Reply by arturmietek** [reply]
Now i also noticed that after few hours its not possible to change state of TS011F_din_smart_relay. I need to disable power by circuit breaker and turn it on again.

**Reply by arturmietek** [reply]
I have 4 of TS011F_din_smart_relay. All of them restarted yesterday. 2 not responding now.

Error 2023-08-03 07:34:10Publish 'get' 'state' to 'Pompa deszczówka' failed: 'Error: Read 0xa4c13806a023879b/1 genOnOff(["onOff"], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Data request failed with error: 'MAC no ack' (233))'
Info 2023-08-03 07:34:10MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'get' 'state' to 'Pompa deszczówka' failed: 'Error: Read 0xa4c13806a023879b/1 genOnOff([\"onOff\"], {\"sendWhen\":\"immediate\",\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":true,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null,\"writeUndiv\":false}) failed (Data request failed with error: 'MAC no ack' (233))'","meta":{"friendly_name":"Pompa deszczówka"},"type":"zigbee_publish_error"}'

But supprise: 1 reports power:
2023-08-03 07:36:51MQTT publish: topic 'zigbee2mqtt/Pompa drenaż', payload '{"current":0,"energy":0.04,"indicator_mode":null,"linkquality":15,"power":16,"power_outage_memory":"restore","state":"OFF","update":{"installed_version":192,"latest_version":192,"state":"idle"},"update_available":false,"voltage":236}'

Today also restarted RPi with corrdinator.

Below one i care about power reading:
Info 2023-08-03 08:00:50MQTT publish: topic 'zigbee2mqtt/Pompa deszczówka', payload '{"current":0,"energy":28.67,"indicator_mode":null,"linkquality":9,"power":0,"power_outage_memory":"restore","state":"ON","update":{"installed_version":192,"latest_version":192,"state":"idle"},"update_available":false,"voltage":236}'

All of them updated to 192 version of firmware.

**Reply by arturmietek** [reply]
Today i click "Apply" button on reporting tab of zigbee2mqtt and that bring me power for 1 device (2 of them still without power and current). Im not sure how reporting configuration works but its seem related.
